packages/framework/src/Component/Image/ImageFacade.php

```diff
@@ -529,4 +542,32 @@ public function getImagesByEntityId(int $id, string $entityClass): array
 
         return $this->getImagesByEntityIdAndNameIndexedById($id, $entityName, null);
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\FileUpload\ImageUploadData $imageUploadData
+     * @param object $entity
+     * @param string|null $type
+     */
+    public function setImageUploadDataForEntityAndType(ImageUploadData $imageUploadData, object $entity, ?string $type = null): void
+    {
+        $orderedImages = $this->getImagesByEntityIndexedById($entity, $type);
+        $imageUploadData->orderedImages = $orderedImages;
+
+        foreach ($orderedImages as $orderedImage) {
+            $imageUploadData->currentFilenamesIndexedByImageIdAndLocale[$orderedImage->getId()] = $orderedImage->getNames();
```

Comment:
```suggestion
            $imageUploadData->namesIndexedByImageIdAndLocale[$orderedImage->getId()] = $orderedImage->getNames();
```
i don't understant why it's named currentFilenames. It's name of image?

Reply:
changed